src/sentry/static/sentry/app/components/globalSelectionHeaderRow.tsx

```diff
@@ -77,7 +77,7 @@ const Content = styled('div')`
 
 const CheckboxHitbox = styled('div')`
   margin: 0 -${space(1)} 0 0; /* pushes the click box to be flush with the edge of the menu */
-  padding: 0 ${space(1.5)} 0 ${space(1.25)};
```

Comment:
we don't have a space(1.25) so originally this line was just evaluated as
`padding: 0 ${space(1.5)} 0;` which is equivalent to just `padding: 0 ${space(1.5)};`